### PR TITLE
src/docs/development/ui/layout/tutorial.md initially completed.

### PR DESCRIPTION
**Pull request by Hao3o5**
src/docs/development/ui/layout/tutorial.md initially completed. 
related issue : https://github.com/cfug/flutter.cn/issues/147.

### CODE REVIEW

File: src/docs/development/ui/layout/tutorial.md [hunk 1]
```diff
@@ -68,17 +96,37 @@ then do the following:
...
 
+## 第一步: 对布局进行图形分解
+
 The first step is to break the layout down to its basic elements:
 
+第一步需要将布局分解成它的各个基础元素：
+
 * Identify the rows and columns.
+
+* 识别出它的行和列。
```
Comment (yanyixin) [inline src/docs/development/ui/layout/tutorial.md]:
```suggestion
  识别出它的行和列。
```

File: src/docs/development/ui/layout/tutorial.md [hunk 2]
```diff
@@ -68,17 +96,37 @@ then do the following:
...
+
 * Does the layout include a grid?
+
+* 这个布局是否包含网格布局？
```
Comment (yanyixin) [inline src/docs/development/ui/layout/tutorial.md]:
```suggestion
  这个布局是否包含网格布局？
```

File: src/docs/development/ui/layout/tutorial.md [hunk 3]
```diff
@@ -68,17 +96,37 @@ then do the following:
...
+
 * Are there overlapping elements?
+
+* 是否有重叠的元素？
```
Comment (yanyixin) [inline src/docs/development/ui/layout/tutorial.md]:
```suggestion
  是否有重叠的元素？
```

File: src/docs/development/ui/layout/tutorial.md [hunk 4]
```diff
@@ -68,17 +96,37 @@ then do the following:
...
+
 * Does the UI need tabs?
+
+* 界面是否需要选项卡？
```
Comment (yanyixin) [inline src/docs/development/ui/layout/tutorial.md]:
```suggestion
  界面是否需要选项卡？
```

File: src/docs/development/ui/layout/tutorial.md [hunk 5]
```diff
@@ -68,17 +96,37 @@ then do the following:
...
+
 * Notice areas that require alignment, padding, or borders.
 
+* 留意需要对齐、内间距、或者边界的区域。
```
Comment (yanyixin) [inline src/docs/development/ui/layout/tutorial.md]:
```suggestion
  留意需要对齐、内间距、或者边界的区域。
```

File: src/docs/development/ui/layout/tutorial.md [hunk 6]
```diff
@@ -34,17 +47,32 @@ and shows how to place a single widget on the screen.
...
+
 Make sure you've [set up](/docs/get-started/install) your environment,
 then do the following:
 
+确保你已经 [设置](/docs/get-started/install) 好了你的环境，
+然后做如下步骤：
+
  1. [Create a basic "Hello World" Flutter app][hello-world].
+
+    [创建一个基础的 "Hello World" Flutter app][hello-world].
```
Comment (yanyixin) [inline src/docs/development/ui/layout/tutorial.md]:
```suggestion
    [创建一个基础的 "Hello World" Flutter app][hello-world]。
```

File: src/docs/development/ui/layout/tutorial.md [hunk 7]
```diff
@@ -34,17 +47,32 @@ and shows how to place a single widget on the screen.
...
+
  2. Change the app bar title and the app title as follows:
 
+    按照如下方法修改　app 标题栏的标题以及 app 的标题：
```
Comment (yanyixin) [inline src/docs/development/ui/layout/tutorial.md]:
```suggestion
    按照如下方法修改 app 标题栏的标题以及 app 的标题：
```

File: src/docs/development/ui/layout/tutorial.md [hunk 8]
```diff
@@ -152,14 +214,28 @@ Widget titleSection = Container(
  1. Putting a Column inside an Expanded widget stretches the column to use all
     remaining free space in the row. Setting the `crossAxisAlignment` property to
     `CrossAxisAlignment.start` positions the column at the start of the row.
+
+    将 Column 元素放到 Expanded widget 中可以拉伸该列，以利用该行中所有剩余的闲置空间。
+    设置 crossAxisAlignment 属性值为 CrossAxisAlignment.start，
```
Comment (yanyixin) [inline src/docs/development/ui/layout/tutorial.md]:
```suggestion
    设置 `crossAxisAlignment` 属性值为 `CrossAxisAlignment.start`，
```

File: src/docs/development/ui/layout/tutorial.md [hunk 9]
```diff
@@ -152,14 +214,28 @@ Widget titleSection = Container(
...
+
+    将第一行文本放入 Container 容器中使得你可以增加内间距。
+    列中的第二个子元素，同样为文本，显示为灰色。
+
  3. The last two items in the title row are a star icon, painted red,
     and the text "41". The entire row is in a Container and padded
     along each edge by 32 pixels.
 
+    标题行中的最后两项是一个红色星形图标，和文字"41"。
+    整行都在一个 Container 容器布局中，而且每条边都有32像素的内间距。
```
Comment (yanyixin) [inline src/docs/development/ui/layout/tutorial.md]:
```suggestion
    整行都在一个 Container 容器布局中，而且每条边都有 32 像素的内间距。
```

File: src/docs/development/ui/layout/tutorial.md [hunk 10]
```diff
@@ -187,25 +263,42 @@ Add the title section to the app body like this:
   - When pasting code into your app, indentation can
     become skewed. You can fix this in your Flutter editor
     using the [automatic reformatting support](/docs/development/tools/formatting).
+
+    在粘贴代码到你的 app 中时，行首缩进可能会发生偏移。
+    你可以通过使用 [代码自动格式化支持](/docs/development/tools/formatting)来修复这个问题。
```
Comment (yanyixin) [inline src/docs/development/ui/layout/tutorial.md]:
```suggestion
    你可以通过使用 [代码自动格式化支持](/docs/development/tools/formatting) 来修复这个问题。
```

Comment (Hao3o5) [reply src/docs/development/ui/layout/tutorial.md]: 感谢，已修正。

**Pull request merged**